### PR TITLE
Print Order Page enchancements

### PR DESCRIPTION
**Pull request by KPull**
1. The full order is now summarized. Each unique item is counted and collected. Each row will contain, for example, "2x Burger" with a list of people who ordered that item.
2. There's now an empty checkbox beside each name so that whoever orders can tick anyone who has already paid.
3. A "Go Back" button on the order page to take the user back to the main page which does not get printed.

### CODE REVIEW

File: webapp/app/views/printOrder.html [hunk 1]
```diff
@@ -6,24 +6,34 @@
...
     <tbody>
-        <tr ng-repeat="item in order.items">
+        <tr ng-repeat="item in order.summary">
             <td>
-                <div class="h3">{{item.name}}</div>
-                <div class="h4">{{item.author}}</div>
+                <div class="h3">{{item.count}} x {{item.name}}</div>
+                <div class="h4 text-muted">
+                    <ul class="list-inline">
+                        <li ng-repeat="author in item.authors"><i class="fa fa-square-o"></i> {{author}}</li>
```
Comment (camilleriluke) [inline webapp/app/views/printOrder.html]: `Duplicates in a repeater are not allowed. Use 'track by' expression to specify unique keys. Repeater: author in item.authors`

Comment (camilleriluke) [inline webapp/app/views/printOrder.html]: How about having reactive checkboxes that will preserve the state?

Two reasons: 
0. This way the user can start collecting money even if the order is not yet printed
1. Pay tracking can be done online without printing - Eco Friendly :seedling: :earth_africa:

Comment (KPull) [inline webapp/app/views/printOrder.html]: Re: repeater error. What's the problem here? How did you get this error to trigger?

Comment (camilleriluke) [inline webapp/app/views/printOrder.html]: Today I feel really hungry and I would like to have 2 avocado salads, so I add the salad twice.

Solution: Add the `trackBy`

**Comment by KPull** [comment]
Completed changes.

**Comment by camilleriluke** [comment]
:+1: gj :dancer:

**Comment by camilleriluke** [comment]
Deployed

**Pull request merged**